mocha.css

```diff
@@ -317,7 +317,7 @@ body {
 
 @media screen and (max-device-width: 480px) {
   #mocha {
-    margin: 60px 0px;
+    margin: 60px 0;
```

Comment:
Prefer to leave 'px' here for documentation

Reply:
It's useless. https://www.w3.org/TR/css-values-3/#lengths